<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatListItem</class>
 <widget class="QWidget" name="ChatListItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>55</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <item>
    <widget class="QWidget" name="profileWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <widget class="QLabel" name="profileLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="redCircleWidget" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>1</y>
        <width>10</width>
        <height>10</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget { 
    border-radius: 5px;
    background-color: red;
}</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="friendNameLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>小鸡毛</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="msgLabel">
         <property name="text">
          <string>你好呀，我是小鸡毛</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timeLabel">
         <property name="text">
          <string>2025/06/16</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
